tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -1290,37 +1295,46 @@ public static MissingStatus getMissingStatus(CLDRFile sourceFile, String path, b
 
         boolean isAliased = !path.equals(status.pathWhereFound); // this was path.equals, which would be incorrect!
-        if (DEBUG
-            && !isAliased
-            && !sourceLocale.equals(sourceLocaleID)) {
-            int debug = 0;
+        if (DEBUG) {
+            if (path.equals("//ldml/characterLabels/characterLabelPattern[@type=\"subscript\"]")) {
+                int debug = 0;
+            }
+            if (!isAliased && !sourceLocale.equals(sourceLocaleID)) {
+                int debug = 0;
+            }
         }
 
         if (value == null) {
-            result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.MISSING_OK : MissingStatus.ABSENT;
+            result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.MISSING_OK
+                : MissingStatus.ABSENT;
         } else {
             /*
              * skipInheritanceMarker must be false for getSourceLocaleIdExtended here, since INHERITANCE_MARKER
              * may be found if there are votes for inheritance, in which case we must not skip up to "root" and
              * treat the item as missing. Reference: https://unicode.org/cldr/trac/ticket/11765
              */
             String localeFound = sourceFile.getSourceLocaleIdExtended(path, status, false /* skipInheritanceMarker */);
+            final boolean localeFoundIsRootOrCodeFallback = localeFound.equals("root")
+                || localeFound.equals(XMLSource.CODE_FALLBACK_ID);
+            final boolean isParentRoot = CLDRLocale.getInstance(sourceFile.getLocaleID()).isParentRoot();
             /*
              * Only count it as missing IF the (localeFound is root or codeFallback)
              * AND the aliasing didn't change the path.
              * Note that localeFound will be where an item with ↑↑↑ was found even though
              * the resolved value is actually inherited from somewhere else.
              */
-            if (localeFound.equals("root")
-                || localeFound.equals(XMLSource.CODE_FALLBACK_ID)) {
-                result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased)
-                    || sourceLocaleID.equals("en")
-                    ? MissingStatus.ROOT_OK
-                        : MissingStatus.ABSENT;
-            } else if (isAliased) {
-                result = MissingStatus.ALIASED;
-            } else {
+
+            if (localeFoundIsRootOrCodeFallback) {
+                result = ValuePathStatus.isMissingOk(sourceFile, path, latin, isAliased) ? MissingStatus.ROOT_OK
+                    : isParentRoot ? MissingStatus.ABSENT
+                        : MissingStatus.ALIASED;
+            } else if (!isAliased) {
                 result = MissingStatus.PRESENT;
+            } else if (isParentRoot) { // We handle ALIASED specially, depending on whether the parent is root or not.
```

Comment:
The main difference in the logic from v39 is that aliases in non-top locales (like en_NZ) are not flagged as ABSENT.